pkg/api/handlers/libpod/volumes.go

```diff
@@ -203,3 +203,20 @@ func RemoveVolume(w http.ResponseWriter, r *http.Request) {
 	}
 	utils.WriteResponse(w, http.StatusNoContent, "")
 }
+
+// ExistsVolume check if a volume exists
+func ExistsVolume(w http.ResponseWriter, r *http.Request) {
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
```

Comment:
I would prefer this to use the function in pkg/domain/infra/abi/volumes.go

We are going to be changing all of the handlers to use abi functions, to make sure podman-remote and other APIV2 tools use the same code as local podman.

Reply:
Ok, understood.